contracts/PearlDistributePrice.sol

```diff
@@ -5,7 +5,7 @@ interface OysterPearl {
     function transfer(address _to, uint256 _value) public;
 }
 
-contract PearlDistribute {
+contract PearlDistributePrice {
```

Comment:
Since you renamed this, the constructor will not longer work. Also not sure if we can rename this since it has already been deployed?

Reply:
neither OysterPearl.sol nor PearlDistributePrice depend on one another, PearlDistributePrice is deprecated.

Reply:
Since this contract is already deployed:
https://etherscan.io/contractsVerified?cn=PearlDistribute

Can we rename the file instead of the contract?

Reply:
Ok, I'll revert it back, and see if I can config truffle instead.